cardano-cli/src/Cardano/CLI/EraBased/Transaction/Option.hs

```diff
@@ -400,21 +401,133 @@ pTransactionCalculateMinReqUTxO era' =
       <$> pProtocolParamsFile
       <*> pTxOutShelleyBased
 
-pTransactionCalculatePlutusScriptCost :: EnvCli -> Parser (TransactionCmds era)
-pTransactionCalculatePlutusScriptCost envCli =
-  fmap TransactionCalculatePlutusScriptCostCmd $
-    TransactionCalculatePlutusScriptCostCmdArgs
-      <$> ( LocalNodeConnectInfo
-              <$> pConsensusModeParams
-              <*> pNetworkId envCli
-              <*> pSocketPath envCli
-          )
-      <*> pTxInputFile
-      <*> optional pOutputFile
+pTransactionCalculatePlutusScriptCost
+  :: ShelleyBasedEra era -> EnvCli -> Parser (TransactionCmds era)
+pTransactionCalculatePlutusScriptCost sbe envCli =
+  ( Opt.hsubparser $
+      commandWithMetavar "online" $
+        Opt.info (pTransactionCalculatePlutusScriptCostParams (pNodeConnectionInfo envCli)) $
+          Opt.progDesc
```

Comment:
suggestion: use leading `.` instead of `$` to vertically align the code with fourmolu

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/1079/commits/ca0ab1fffb1b78353424921a47c30bdee50f9b56